explanation/intro-to/ebpf.md

```diff
@@ -0,0 +1,134 @@
+(introduction-to-ebpf)=
+# Introduction to eBPF
+
+[eBPF](https://ebpf.io/) is a powerful tool for server and system
+administrators. It provides the ability to extract plenty of contextual
+information from systems or processes, offering several capabilities for
+monitoring, debugging, and securing systems with little overhead given it runs
+in kernel-space, i.e., there is no need for context-switching, making it very
```

Comment:
I applied this change.  I have only used "on the other hand" to express ideas conflicting with the first one. This is the first time I see it applied to express a complimentary idea :)

Reply:
Well, let me check with @s-makin if that is OK to be done that way :-)

it is usually more "this option has benefit A, the other has benefit B"
But it is defined as:
"used when you are comparing two different facts or two opposite ways of thinking about a situation"

So to me (might be wrong) it is also ok to say.
"Because of fact A it has benefit A', and because of fact B it has benefit B'"
as in
"On one hand due to fact A it has benefit A', and on the other hand because of fact B it has benefit B'"

@s-makin if this is stupid or even enough to be misleading it is easy to fix without involving Athos, let me know.

Reply:
Sorted out as a team \o/